### ISSUE TITLE
>status not working

### ISSUE THREAD

**Issue by TISnoob** [finlaysawyer/discord-uptime#55]

**Comment by glitch144** [comment]
Im having this issue too any fixes?

**Comment by DarkMaTTTTTTer** [comment]
Same

**Comment by glitch144** [comment]
Hey guys, 
i managed to fix it by changing the prefix to something else. I am not sure if its related to this issue or not but i got this error
``` File "~discord-uptime-master/cogs/monitor.py", line 118, in notify_up
    self.retry_count.pop(server["address"])
KeyError: '<your ip>' 
I removed line 
 self.retry_count.pop(server["address"])  and it worked for me

**Comment by finlaysawyer** [comment]
Given the deprecation of discord.py, I have no immediate plans to update this project. I will accept PRs if appropriate

**Comment by finlaysawyer** [comment]
This should be fixed in #74 / #76
